_posts/2016-09-22-foreman-community-newsletter---september-2016.md

```diff
@@ -0,0 +1,115 @@
+
+The latest point release of the Foreman 1.12 series is now available, bringing
+bugfixes and support for Puppet Server 2.6. You can read all the details in the
+[announcement][1_12_3]
+
+[1_12_3]: https://groups.google.com/d/topic/foreman-users/0njkhkpRWzM/discussion
+
+### EL6 deprecated in 1.13, to be removed in 1.14
+
+Users of EL6 / Centos6 should be aware that 1.13 will be the last release with
+packages for this OS. 1.13 will be supported until early next year, and users
+are strongly encouraged to start planning their migrations before that date.
+
+Note that installation of EL6 hosts from Foreman is still supported - it's the
+packages for Foreman itself that will no longer be produced for EL6.
+
+### Katello 3.1 and 3.2 RC1
+
+The long-awaited Katello 3.1 release happened on September 9th, bringing a
+welcome round of improvements. You can learn more via the
+[announcement][K_3_1].
+
+Hot on 3.1's heels, Katello 3.2 RC1 has just been released - notably bring
+support for Puppet 4 to Katello. As always, your testing is hugely appreciated
+in making these releases as stable as possible - if you'd like to try it out,
+check out [this thread][K_3_2_rc1] and [let us know][kbugs] how it goes.
+Thanks!
+
+[K_3_1]: https://groups.google.com/d/topic/foreman-users/jnvGiYYpd9A/discussion
+[K_3_2_rc1]: https://groups.google.com/d/topic/foreman-users/3gGorphv1vI/discussion
+[kbugs]: http://projects.theforeman.org/projects/katello/issues/new
+
+### New media and blogs
+
+#### [Community Demo - 15th September](https://youtu.be/GFNPHypFhl4)
+
+Another packed demo, great to see them building up again. This time we had some
+Roles/Orgs updates, Ansible & Remote Execution integration, Katello
+improvements and a new MS-DHCP provider for the smart proxy. Check it out!
+
+### Upcoming events
+
+#### [AnsibleFest Brooklyn - 11th October](https://www.ansible.com/ansiblefest)
+
+We're hoping to have a small presence at the next AnsibleFest in Brooklyn next
+month - if you're planning to go along, [get in touch][mail_greg] and if
+there's enough interest, perhaps we can get a Foreman/Ansible thing going on
+the day!
+
+[mail_greg]: mailto:greg.sutcliffe@gmail.com
+
+#### [Community Demo - 13th October](https://youtu.be/Rao1O1mc8AU)
+
+The next community demo will be on 13th October, and as ever you're warmly
+welcomed to join us live for an hour of discussion on the latest work done in
+the Foreman community. See you then!
+
+#### [CfgMgmtCamp Berlin - 15th November](http://cfgmgmtcamp.eu/berlin-2016)
+
+If you love Config Management Camp in Ghent, but just can't wait until
+February, then you'll want to check out the one day event being organised in
+Berlin. Registration *and* the CFP are still open, so book in, and maybe even
+submit a Foreman talk of your own!
+
+_(All Foreman events are now available via the [Foreman Calendar](/events))_
+
+### Updated plugins
+
+- [foreman_bootdisk](https://github.com/theforeman/foreman_bootdisk) updated to 8.0.2
```

Comment:
Foreman Ansible had a pretty important update where running roles playbooks is now orchestrated by foreman-tasks, it's waiting on https://github.com/theforeman/foreman-packaging/pull/1335 and I'll make an announcement on the mailing list once it's merged